frontend/packages/dev-console/src/actions/add-resources.tsx

```diff
@@ -93,5 +102,6 @@ export const addResourceMenuWithoutCatalog: KebabAction[] = [
   fromGit,
   containerImage,
   fromDockerfile,
+  fromDevfile,
```

Comment:
We should add `fromDevfile` to `addGroupResourceMenu` as well
<img width="616" alt="Screenshot 2020-12-03 at 11 54 58 AM" src="https://user-images.githubusercontent.com/9278015/100972810-6b1a8f00-355f-11eb-919d-d6485842229b.png">